source/MaterialXRuntime/Private/PvtApi.h

```diff
@@ -292,6 +307,70 @@ class PvtApi
         return _implementationSearchPaths;
     }
 
+    const FileSearchPath& getMaterialXDefinitionPaths() const
+    {
+        return _materialXDefinitionPaths;
+    }
+
+    const FileSearchPath& getGlobalDefinitionPaths() const
+    {
+        return _globalDefinitionPaths;
+    }
+
+    const FileSearchPath& getLocalDefinitionPaths() const
+    {
+        return _localDefinitionPaths;
+    }
+
+    const FileSearchPath& getMaterialXCoreDefinitionPaths() const
+    {
+        return _materialXCoreDefinitionPaths;
+    }
+
+    const FileSearchPath getDefinitionPaths(bool includeSubFolders=true) const
```

Comment:
I don't see this being used anywhere ?
The location of the definitions is also part of where image should be searched for.

Reply:
I'm not currently using this anywhere. Should it be used as the library path passed to VnnRuntime::initRuntime?